docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,293 @@
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
+
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            you@laptop:~$ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Ubuntu / Debian
+         ^^^^^^^^^^^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \         
+              virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
+
+         Fedora 20
+         ^^^^^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv && \
+              virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Live
+
+      .. noscript-content::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
+
+         .. code-block:: console
+
+            you@laptop:~$ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
+
+         .. code-block:: console
+
+            you@laptop:~$ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
```

Comment:
Please file a ticket for de-duplicating the content that appears both here and under the `noscript-content` block.